vision.py

```diff
@@ -177,18 +198,20 @@ def get_image_values(self, frame: np.ndarray) -> tuple:
         self.mask = cv2.inRange(
             self.hsv, HSV_LOWER_BOUND, HSV_UPPER_BOUND, dst=self.mask
         )
-        self.mask = cv2.erode(self.mask, None, dst=self.mask, iterations=1)
-        self.mask = cv2.dilate(self.mask, None, dst=self.mask, iterations=2)
-        self.mask = cv2.erode(self.mask, None, dst=self.mask, iterations=1)
+        # self.mask = cv2.erode(self.mask, None, dst=self.mask, iterations=1)
+        # self.mask = cv2.dilate(self.mask, None, dst=self.mask, iterations=2)
+        # self.mask = cv2.erode(self.mask, None, dst=self.mask, iterations=1)
```

Comment:
Why are these commented out?

Reply:
You shouldn't just resolve without answering the question here, even if you talked to Lucien about it. Otherwise the record here is incomplete.